src/Utils/queryUtils.js

```diff
@@ -10,13 +12,17 @@ export function queryForID(data, id) {
 
 export function queryForCollection(data, body) {
   let query = data;
-  const { limit, filters } = body;
+  const { limit, filters, order } = body;
   if (filters) {
     query = applyFilters(query, filters);
   }
   if (limit) {
     query = query.limit(Number(limit));
   }
+  if(order) {
+    const {orderBy, orderDirection} = order;
```

Comment:
I think the orderDirection should be apply always. You may want to set a direction without ordering by something in particular.
```
return query.orderBy(orderDirection || 'asc').get().then(snapshot => snapshot.docs.map(doc => ({ ...doc.data(), id: doc.id })));
```

Reply:
Maybe not, you don't want any direction in particular and this could add a boilerplate to the query and some overhead too

Reply:
You are right frank, well i think it should remain like this then